### DISCUSSION TITLE
Disable onboarding for our organization

### DISCUSSION THREAD

**Discussion by pietrodn** [renovatebot/renovate#28522] [Request Help] [answered]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

GitHub

### Please tell us more about your question or problem

Hello, similar to #18826 we would like to have onboarding disabled for the `meetup` organization (never run Renovate unless we push a config in every repo).

Thanks!

**Comment by rarkins** [comment, marked as answer]
What is your current installation status? e.g. not yet installed?

The Mend Renovate app today has this behavior for new installs:
- If you install into "All repositories" then it will default to _Silent_ mode, which means dryRun=lookup. You won't get onboarding or PRs even with a manual repository config
- If you install into "Select repositories" then it will default to _Interactive_ mode, i.e. onboarding

If not yet installed, I believe the following will achieve what you want:
- Install github.com/apps/renovate into All repositories (will default to silently scanning each repo in dry run mode)
- Go to https://developer.mend.io/github/meetup/-/settings, select the Renovate menu, then change from Silent to Interactive, plus "Scan repos with config file"

It should look like this:

![image](https://github.com/renovatebot/renovate/assets/6311784/bfbd530b-33bc-4bce-bfd1-92a97fbe9501)

Alternatively:
- Keep the default Silent mode for the org
- Tell developers that they can trigger onboarding for individual repositories by logging into developer.mend.io, navigating to the repo settings, then changing from Silent to Interactive (which triggers onboarding for that repo only)

**Reply by pietrodn** [reply]
Perfect. I never want to get onboarding so the default settings are fine. Thank you!
